### DISCUSSION TITLE
SVG of panel layout

### DISCUSSION THREAD

**Discussion by mockendon** [kachurovskiy/nanoels#35] [Show and tell]
In case it helps someone, here is an svg layout that can be used for custom design and drag knife toolpath to cut it out on your cnc.  The gcode xy origin is at the the register on the SVG.

https://drive.google.com/drive/folders/1hQHsJYtmsrNX-O1yQpdYYvHm5j1s70us?usp=sharing

**Comment by kachurovskiy** [comment]
That's nice, thank you! Which drag knife machine do you have?

**Reply by mockendon** [reply]
I just put a Roland style vinyl dragknife on my cheap Chinese CNC.  Works great.

**Comment by ws786873** [comment]
@mockendon Hello, I see your great drawing, but I use a separate control box. Can you help draw a corresponding drawing? I'm going to find a factory to make PVC panels

**Reply by mockendon** [reply]
Sure, what do you need?

**Reply by ws786873** [reply]
> 当然，你需要什么？
<img width="743" alt="case" src="https://user-images.githubusercontent.com/25994407/150683590-59364eb8-5a8a-4913-a593-a64a7750c810.png">
i need a plane for this case,thank you~

**Reply by ws786873** [reply]
i think it should be a SVG file?

**Reply by mockendon** [reply]
Hope this helps.  BTW I just flattened the STL to get the layout so will be 1:1 with the edge of the enclosure as printed.

https://drive.google.com/file/d/1hjgeb3DxMN-lar_S1WoxeJuKDBaw2kgB/view?usp=sharing

**Reply by kachurovskiy** [reply]
Thank you, Matt! Printed scaled 134% and it fits well:

![image](https://user-images.githubusercontent.com/517919/150698955-edb33fa4-9732-409c-a51b-c11b884ca9b9.png)

**Reply by ws786873** [reply]
> Hope this helps. BTW I just flattened the STL to get the layout so will be 1:1 with the edge of the enclosure as printed.
> 
> https://drive.google.com/file/d/1hjgeb3DxMN-lar_S1WoxeJuKDBaw2kgB/view?usp=sharing

it works,scale is ok
